packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxisRendering.selectors.ts

```diff
@@ -53,6 +53,11 @@ export const createZoomMap = (zoom: readonly ZoomData[]) => {
 const selectorChartZoomState = (state: ChartState<[], [UseChartCartesianAxisSignature]>) =>
   state.zoom;
 
+export const selectorChartHasZoom = createSelector(
+  [selectorChartZoomState],
+  (zoom) => (zoom?.zoomData?.length ?? 0) > 0,
```

Comment:
Not sure we should use `zoomData` or `selectorChartZoomOptionsLookup`, which would check if the axes are configured for this, probably `zoomData` works as we initialise it when zoom is enabled.